### ISSUE TITLE
Master branch version is behind 1.15.x branch version

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/Products.Archetypes#130]
TLDR: The history in `CHANGES.rst` is messed up. I am preparing a PR as I write this issue.

Jenkins 5.2 Python 2.7 was [failing](https://jenkins.plone.org/job/plone-5.2-python-2.7/84/testReport/) in three Archetypes tests.
Direct cause was [this coredev commit](https://github.com/plone/buildout.coredev/commit/b4cb868999a384b92d9201c5d86b648fe0e476de) by @esteele where he released Products.Archetypes 1.15.6, which replaced the 1.16.0 that was there.
I have added Products.Archetypes to the checkouts again, and that fixes the tests (at least locally).

But that commit points to an underlying problem: the master branch is behind the 1.15.x branch.

We have these tags in git:
- 1.15.6 (2018-12-10), from commit 209b769b79692c7b97e2d4f7af163e017da64d41 on branch 1.15.x
- 1.15.7 (2019-01-09), from commit a7b3b358ee1169756e4f482163424bfe8325d374 on branch 1.15.x
- 1.16.0 (2018-12-10), from commit c72a61008b44222ee4f9bdf91a4d043f9a811d1a, which isn't on any branch

Some other notes:

- The `CHANGES.rst` of both branches miss an entry for release 1.15.5, which does exist in the [tags](https://github.com/plone/Products.Archetypes/releases/tag/1.15.5). Likely some problem at the time with towncrier, as the tag commit 865af4384dca69b2b950da545119c9295699d542 only contains a change to the version number in `setup.py`.
- Only the master branch contains commit 82e2695c9b186e39d133157c703bfa21646de5ea which is a fix for one of the test failures on Jenkins.
- Tag 1.15.5 contains commit 67d280d8883d960dfb8493c7c8108d36b4d5a512 which was only meant for Plone 5.2. The master branch contains this. Tag 1.15.6 does not contain it, and the 1.15.x branch does not contain this commit either.
- So branch 1.15.x was created at some point before that commit, even though tag 1.15.5 was already released with the commit. So it seems a bit messy, and hard to reconstruct.

From what I can see, the branches do seem to contain the code that they should. So the only thing that seems to need a change, is the changelog. I am doing that and will create a PR.

cc @plone/release-team

**Comment by mauritsvanrees** [comment]
Release 1.16.1 has the fixed changelog and all the other real fixes from master.

**Comment by mauritsvanrees** [comment]
I have fixed the changelog in 1.15.x too, and released 1.15.8.
